xdsl/utils/parse_pipeline.py

```diff
@@ -106,6 +106,22 @@ def peek(self) -> Token:
 PassArgListType = list[PassArgElementType]
 
 
+def _pass_arg_element_type_str(arg: PassArgElementType) -> str:
+    if isinstance(arg, bool):
+        if arg:
+            return str(arg).lower()
+        else:
+            return str(arg).lower()
+    elif isinstance(arg, str):
+        return f'"{arg}"'
+    else:
+        return str(arg)
+
+
+def _pass_arg_list_type_str(arg: PassArgListType) -> str:
+    return ",".join(_pass_arg_element_type_str(val) for val in arg)
```

Comment:
Could you please add the name logic in here? Will make the code below a little simpler
```suggestion
def _named_pass_arg_list_type_str(name: str, arg: PassArgListType) -> str:
    if arg:
        return f'{name}={",".join(_pass_arg_element_type_str(val) for val in arg)}"
    else:
        return name
```